x/auth/module.go

```diff
@@ -151,6 +151,10 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	if err := cfg.RegisterMigration(types.ModuleName, 4, func(ctx sdk.Context) error { return nil }); err != nil {
 		panic(fmt.Sprintf("failed to migrate x/%s from version 4 to 5: %v", types.ModuleName, err))
 	}
+
+	if err := cfg.RegisterMigration(types.ModuleName, 5, m.Migrate5to6); err != nil {
```

Comment:
This is actually migration 4 to 5, as migration 4 was for v0.47. You can as well delete the placeholder for migration to 5, which wasn't necessary (https://github.com/cosmos/cosmos-sdk/pull/14483#discussion_r1062294196)